CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 - Add `BindReturn` to the `property` CE ([#364][364], [@TysonMN][TysonMN])
   - A breaking change.  Previously, returning a `bool` from a `property` CE (after using `let!`) caused the CE to have return type `Property<unit>`.  Now this results in a return type of `Property<bool>`.  The previous behavior can now be expressed by piping the `Property<bool>` instance into `Property.falseToFailure`.
 - Change recheck API to accept recheck data encoded as `string` ([#385][385], [@TysonMN][TysonMN])
+- Add RecheckTypeData to simplify recheck reporting ([#386][386], [@TysonMN][TysonMN])
```

Comment:
RecheckTypeData should be backticked for consistency

Reply:
Done